server/ansible/roles/pbench-clean-yum-cache/tasks/main.yml

```diff
@@ -1,10 +1,12 @@
 ---
```

Comment:
Is there a way to make this role common or perhaps see if there is a galaxy role already that does this?

Reply:
I don't know if there is one already, but I am planning to make it common when moving this stuff up to Galaxy. There is a bunch of duplication between server and agent that is going to go away then.